web/package.json

```diff
@@ -10,7 +10,7 @@
     "format": "prettier --write .",
     "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
     "lint:css": "stylelint \"**/**.{css,scss,sass}\"",
-    "lint": "yarn lint:js lint:css",
+    "lint": "yarn run lint:js && yarn run lint:css",
```

Comment:
Could you please delete the change on package.json? This issue will be solved in #7 as well.